bzt/modules/aggregator.py

```diff
@@ -69,8 +69,8 @@ def __init__(self, perc_levels=(), rt_dist_maxlen=None):
         self.get(self.BYTE_COUNT, 0)
         # vectors
         self.get(self.ERRORS, [])
-        self.get(self.RESP_TIMES, Counter())
         self.get(self.RESP_CODES, Counter())
+        self.get(self.RESP_TIMES, HdrHistogram(1, 60 * 30 * 1000, 3))  # is maximum value of 30 minutes enough?
```

Comment:
lowest trackable value can round down to zero, you have 1 as minimum

Reply:
It's a specific of HDR histograms. The constructor should specify minimal non-zero value that should be distinguished. Zeroes will still be counted.